scripts/irods/test/test_iadmin.py

```diff
@@ -1708,6 +1708,36 @@ def test_iadmin_exits_nonzero_on_nonexistent_command__issue_7734(self):
         ec, _, _ = self.admin.assert_icommand(['iadmin', 'nonexistentcommand'], 'STDERR');
         self.assertNotEqual(ec, 0)
 
+    def test_iadmin_rmuser_removes_associated_data__issue_7778(self):
+        test_user_name = 'test_user_issue_7778'
+        test_group_name = 'test_group_issue_7778'
+        test_collection_name = 'test_collection_issue_7778'
```

Comment:
Should we add a test (or add to this test) for data objects as well? I know from the implementation that it "should" work, but, y'know... "blackbox testing" and all that...

Reply:
Sure, I can look into that. More test cases = better, right?